src/ordered-elements.js

```diff
@@ -1,5 +1,4 @@
 /* @flow */
-/* global Map */
 
 export default class OrderedElements {
```

Comment:
Figured it out. The constructor was never called with arguments, so I removed the args.